src/cli-constant.js

```diff
@@ -231,6 +231,7 @@ const detailedOptions = normalizeDetailedOptions({
       "less",
       "scss",
       "json",
+      "glimmer",
```

Comment:
Just comment this line out and update snapshots for now. When we release it we'll uncomment.